test/dummy/app/assets/javascripts/modals.coffee

```diff
@@ -92,6 +92,31 @@ $(document).on 'click', 'a.modal-reload', ->
 		title_icon: 'gear-a'
 	)
 
+## On Show
+
+$(document).on 'click', 'a.modal-onshow', ->
+	closeAction = { title: 'Close', class: 'close-modal' }
+	openNewModal = ->
+		tinyModal.showDirect(
+			"<h1 class=text-center'>Close this modal to execute the callback associated with the previous one!</h1>"
+			title: 'New Modal'
+			actions: [ closeAction ]
+		)
```

Comment:
I would inline this to more clearly indicate that the new modal is kinda nested within the first modal.